Evol du gabarit ArticleObservatoire (RS-1786)

sguellec opened the pull request:
Adaptations graphiques suite modif DS.
Ajout du champ "contenu"

sguellec committed:
Evol du gabarit ArticleObservatoire (RS-1786)

Adaptations graphiques suite modif DS.
Ajout du champ "contenu"

LChoquet commented on plugins/ObservatoirePlugin/types/ArticleObservatoire/doArticleObservatoireFullDisplay.jsp:
```diff
@@ -65,31 +70,42 @@
+	                        <ul class="ds44-list ds44-list--tag ds44--m-padding">
 								<jalios:foreach collection="<%= obj.getMotsCles(loggedMember) %>" type="Category" name="itCategory">
-									<li><%=itCategory%></li>
+									<li><a href="#" class="ds44-btnStd ds44-btn--invert ds44-btnStd--tag" title='<%= glp("jcmsplugin.observatoire.motcle.title", itCategory)%>'><span class="ds44-btnInnerText"><%=itCategory%></span></a></li>
```
C'est normal le href sur # ?

sguellec replied:
Oui, ça pointera vers une URL de recherche. Mickael travaille sur le sujet en ce moment.

LChoquet commented on plugins/ObservatoirePlugin/types/ArticleObservatoire/doArticleObservatoireFullDisplay.jsp:
```diff
@@ -25,6 +25,11 @@
                     </jalios:if>
                     
+                    <%-- Contenu --%>
+                    <jalios:if predicate='<%= Util.notEmpty(obj.getContenu()) %>'>
```
Un petit "userLang" pour le multilingue ? S'il y a

sguellec replied:
Ce site ne sera pas en multilingue mais c'est en effet une bonne pratique.

LChoquet reviewed (approved):
OK pour moi avec le userLang dans la possibilité du multilingue

sguellec committed:
Fix : multilingue

Appel des champs texte en multilingue pour uniformiser, même s'il n'est pas prévu de version anglaise pour ce site.
